### PR TITLE
UI Changes and Improvements + ID to Data-CY 

### PR DESCRIPTION
**Pull request by StapleMink**

### CODE REVIEW

File: cypress/cypress/integration/lessons.spec.ts [hunk 1]
```diff
@@ -97,11 +97,11 @@ describe("lessons screen", () => {
     cy.visit("/lessons");
     cy.get("#column-header");
     cy.get("#column-header #name").contains("Lesson");
-    cy.get("#column-header #launch").contains("Launch");
-    cy.get("#column-header #grade").contains("Grade");
+    // cy.get("#column-header #launch-button").contains("Launch");
```
Comment (beatthat) [inline cypress/cypress/integration/lessons.spec.ts:100]: if we're getting rid of these, just delete...double check we actually don't need this coverage (e.g. there's a test for visibility of this button elsewhere?)

**Review by beatthat — CHANGES_REQUESTED** [review]
Looking good. There's a few things:

1. in places where we're deleting test lines, just delete don't comment--but make triple sure first we don't actually want some comparable test there?

2. Coordinate with @supersweet-dev and make sure he's aware of the changing HTML `id` names because he's going to probably have conflict w the `data-cy` change he's working on in parallel

3. Some of the snapshots look definitely wrong (e.g. [thos one](https://github.com/opentutor/opentutor-admin/blob/a7770b8499cc6872f7fb80997d5a4feea09153a8/cypress/cypress/snapshots/screenshots-grade-session.spec.ts/screenshots-grade-session-displays-feedback-after-answer-marked-good.snap.png)). We should decide on a case-by-case basis if we can make it work or need to abandon

File: cypress/cypress/integration/lessons.spec.ts (hunk 1 above)
Comment (StapleMink) [reply cypress/cypress/integration/lessons.spec.ts]: Fixed

File: client/src/pages/lessons/index.tsx [hunk 2]
```diff
@@ -212,52 +195,62 @@ const LessonItem = (props: {
...
-        >
-          <DeleteIcon />
-        </IconButton>
-      </TableCell>
-      <TableCell id="copy" align="center">
-        <IconButton onClick={handleCopy}>
-          <FileCopyIcon />
-        </IconButton>
+      <TableCell data-cy="actions" align="right">
+        <Tooltip title="Grade" arrow>
```
Comment (beatthat) [inline client/src/pages/lessons/index.tsx:218]: did you add `cypress` tests for the new tool tips here?

**Review by beatthat — CHANGES_REQUESTED** [review]
looks great. Only change request is let's get tests for the new tool tips, e.g. `shows tooltip on mouseover XXX`

**Review by beatthat — APPROVED** [review]

**Pull request merged**